connectors/PartnerCenterReferrals/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1097 @@
+            "name": "X-Caller-Id",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "properties": {
+                "@odata.context": {
+                  "description": "@odata.context",
+                  "title": "ODataContext",
+                  "type": "string"
+                },
+                "@odata.count": {
+                  "description": "@odata.count",
+                  "format": "int32",
+                  "title": "ODataCount",
+                  "type": "integer"
+                },
+                "value": {
+                  "items": {
+                    "$ref": "#/definitions/Microsoft.Partner.Service.PartnerReferrals.Contracts.V3.Referral"
+                  },
+                  "type": "array"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "There was a missing or an invalid input. The response body will provide the error details."
+          },
+          "401": {
+            "description": "The request was not authenticated. The client needs to authenticate with the partner API service first."
+          },
+          "403": {
+            "description": "The request was authenticated but was refused since the caller does not have the rights to invoke it."
+          },
+          "404": {
+            "description": "The resource is not found."
+          },
+          "500": {
+            "description": "The partner API service or one of its dependencies failed to fulfill the request. Callers may retry this."
+          }
+        },
+        "summary": "Get all referrals",
+        "tags": [
+          "Referrals"
+        ]
+      },
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "deprecated": false,
+        "description": "Create a referral",
+        "operationId": "CreateReferral",
+        "parameters": [
+          {
+            "description": "The entity to post",
+            "in": "body",
+            "name": "Referral",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Microsoft.Partner.Service.PartnerReferrals.Contracts.V3.Referral"
+            }
+          },
+          {
+            "description": "The unique identifier to track the request",
+            "in": "header",
+            "name": "MS-CorrelationId",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Correlation Identifier",
+            "x-ms-visibility": "important"
+          },
+          {
+            "default": "60858c1b-2062-40ad-b42c-996b6b5b047d",
+            "description": "Optional, ID to identify the caller - PowerAutomate",
```

Comment:
@didalal Please add a space between Power and Automate like below:

`"description": "Optional, ID to identify the caller - Power Automate",`